plugins/devtools-backend/README.md

```diff
@@ -49,6 +49,34 @@ Here's how to get the DevTools Backend up and running:
 4. Now run `yarn start-backend` from the repo root
 5. Finally open `http://localhost:7007/api/devtools/health` in a browser and it should return `{"status":"ok"}`
 
+### New Backend System
+
+The DevTools backend plugin has alpha support for the [new backend system](https://backstage.io/docs/backend-system/), here's how you can set that up:
+
+In your `packages/backend/src/index.ts` make the following changes:
+
+```diff
+  import { createBackend } from '@backstage/backend-defaults';
+  import { appPlugin } from '@backstage/plugin-app-backend';
+  import { catalogPlugin } from '@backstage/plugin-catalog-backend';
+  import {
+    scaffolderPlugin,
+    catalogModuleTemplateKind,
+  } from '@backstage/plugin-scaffolder-backend';
++ import { devtoolsPlugin } from '@backstage/plugin-devtools-backend/alpha';
```

Comment:
In all these instructions, would it make sense to make them even more compact down to the essentials? I know some people are helped by extra context but some are also confused by it ("my file doesn't have the appPlugin line, is that necessary?"). Could perhaps shorten to something like

```ts
import { devtoolsPlugin } from '@backstage/plugin-devtools-backend';

const backend = createBackend();
backend.add(devtoolsPlugin());
// ... other feature additions

backend.start();
```

or so?

Reply:
Makes sense to me, done 👍